### PR TITLE
SONiC Entity MIB Extension HLD

### PR DESCRIPTION
**Pull request by keboliu**

### CODE REVIEW

File: extension-to-phsical-entity-mib.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+ | Rev |     Date    |       Author       | Change Description                |
+ |:---:|:-----------:|:------------------:|-----------------------------------|
+ | 0.1 |             |      Kebo Liu      | Initial version                   |
+
+
+
+## 1. Overview 
+
+The Entity MIB contains several groups of MIB objects, currently SONiC only implemented part of the entityPhysical group following RFC2737. The definition of "entityPhsical" group defined below.
```
Comment (Junchao-Mellanox) [inline extension-to-phsical-entity-mib.md]: entityPhsical -> entityPhysical

File: extension-to-phsical-entity-mib.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	SNMPv2-SMI::mib-2.47.1.1.1.1.2.1032 = STRING: "DOM TX Bias Sensor for Ethernet0/3"
+	SNMPv2-SMI::mib-2.47.1.1.1.1.2.1033 = STRING: "DOM TX Power Sensor for Ethernet0/3"
+	SNMPv2-SMI::mib-2.47.1.1.1.1.2.1041 = STRING: "DOM RX Power Sensor for Ethernet0/4"
+	SNMPv2-SMI::mib-2.47.1.1.1.1.2.1042 = STRING: "DOM TX Bias Sensor for Ethernet0/4"
+	SNMPv2-SMI::mib-2.47.1.1.1.1.2.1043 = STRING: "DOM TX Power Sensor for Ethernet0/4"
+
+## 3. New extension to Entity MIB implementation
+This extension aim to implement all the objects in entityPhysical group.
+
+Also plan to add more physical entities such as thermal sensors, fan, and it's tachometers, PSU, PSU fan, and some sensors contained in PSU.
```
Comment (Junchao-Mellanox) [inline extension-to-phsical-entity-mib.md]: Should we add LED to the MIB?

Comment (keboliu) [reply extension-to-phsical-entity-mib.md]: No, LED not in the scope.

File: extension-to-phsical-entity-mib.md (hunk 1 above)
Comment (keboliu) [reply extension-to-phsical-entity-mib.md]: fixed.

**Pull request merged**